lawnchair/res/values/config.xml

```diff
@@ -32,6 +32,12 @@
         <item>org.withouthat.acalendar</item>
         <item>ws.xsoh.etar</item>
     </string-array>
+    <string-array name="themed_icon_packs" translatable="false">
```

Comment:
Intrestingly lawnchair works well icon packs that were made from svg with trasparanet background. Found below icon packs works well. So added here let me know if this can be added

Reply:
While this is great, having to keep a static list of packages up to date can be very hard, especially since there are tons of icon packs. If we manage to allow users to modify the list on runtime it would be much better, especially for the maintainers.

Reply:
Good idea, so we need a selector to select icon packs from apps list?

Reply:
But one problem is icons might get break if themed enabled for not supported icon pack. 

If this can be decided by user then we don't need this check whether packages installed to enable _themed icons_ option .

So user will decide if they can enable  _themed icons_  options for the icon pack they selected

Reply:
If themed enabled for unsupported icon pack
![WhatsApp Image 2022-11-08 at 4 31 23 PM](https://user-images.githubusercontent.com/7315975/200548009-d9d36a79-28a3-4ae7-87f0-c113f5f4b146.jpeg)

in this can user need to turn off themed option

Reply:
How about having static list with lawnchair and whatever tested will be added here with PR. Having an advanced option, may be in experimetal feature that enables it for all icon packs too.

Reply:
We can warn the user that not all icon packs support this feature. We just have to make sure it's clear enough so we don't get tons of messages about issues.

About having a static list, I strongly disagree with it at least if it's locally stored. Because not only do we have to update the list on the code, we have to push updates frequently just to let users use the icon packs they want. Basically a bad experience for both sides.

I think the final feature would be ideal if:
- The local static list can be updated from a remote source.
- The user can select any icon pack to be used with proper warnings etc but it probably should still be an experimental feature for more advanced users.

Although... this does sound like lots of more work, especially on the UI UX aspects. Maybe it would be better if we just merge the current PR with the core feature and then add additional QoL improvements with time.